### PR TITLE
Fix transmission -> webrtc

### LINKED ISSUE #25223: configuration type "transmission" should be replaced with "webrtc"
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/MediaCapabilities/encodingInfo

### What specific section or headline is this issue about?

Parameter type "transmission" of configuration object should be replaced

### What information was incorrect, unhelpful, or incomplete?

The value "transmission" of parameter "type" of configuration object should be replaced with "webrtc".

### What did you expect to see?

The value "transmission" should be "webrtc".

### Do you have any supporting links, references, or citations?

This is what the spec is saying: https://w3c.github.io/media-capabilities/#mediaencodingtype

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/mediacapabilities/encodinginfo`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/MediaCapabilities/encodingInfo
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/mediacapabilities/encodinginfo/index.md
* Last commit: https://github.com/mdn/content/commit/2ba2c0efbdf0c34b1da02203e4e84b571c883629
* Document last modified: 2023-02-28T14:00:22.000Z

</details>

### PR DESCRIPTION
**Pull request by teoli2003**
Fixes #25223

(Note that this also matches what is written in `decodingInfo()`)

### CODE REVIEW

File: files/en-us/web/api/mediacapabilities/encodinginfo/index.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ encodingInfo(configuration)
 
         - `record`
           - : Represents a configuration for recording of media, e.g. using {{domxref("MediaRecorder")}}.
-        - `transmission`
-          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}).
+        - `webrtc`
+          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instad.
```
Comment (teoli2003) [inline files/en-us/web/api/mediacapabilities/encodinginfo/index.md:32]:
```suggestion
          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instead.
```

Comment (giftcardvalidator) [reply files/en-us/web/api/mediacapabilities/encodinginfo/index.md]: instead

**Review by wbamberg — COMMENTED** [review]
I thought that as a rule we try to keep browser-specific things inside BCD?

Even if we do want to keep this in the page, we should definitely document it in BCD.

**Comment by teoli2003** [comment]
Note: added to bcd.

File: files/en-us/web/api/mediacapabilities/encodinginfo/index.md [hunk 2]
```diff
@@ -28,8 +28,10 @@ encodingInfo(configuration)
 
         - `record`
           - : Represents a configuration for recording of media, e.g. using {{domxref("MediaRecorder")}}.
-        - `transmission`
-          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}).
+        - `webrtc`
+          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** doesn't work on Firefox, uses `transmission` instead.
```
Comment (wbamberg) [inline files/en-us/web/api/mediacapabilities/encodinginfo/index.md:32]:
```suggestion
          - : Represents a configuration meant to be transmitted over electronic means (e.g. using {{domxref("RTCPeerConnection")}}). **Note:** Firefox uses `transmission` for this type, and `webrtc` does not work.
```

**Review by wbamberg — APPROVED** [review]
👍

**Pull request merged**